reframe/frontend/cli.py

```diff
@@ -81,6 +84,12 @@ def main():
         help='Load checks recursively')
 
     # Select options
+    select_options.add_argument(
+        '--modules-mapping', action='store', dest='module_map_file',
+        help='Map a module name to a different one')
+    select_options.add_argument(
+        '--try-module', action='append', dest='module_map_cl', default=[],
```

Comment:
@victorusu @teojgo Do you agree with the `--try-module` name of the option?

@rsarm The `dest` name should have a better name, e.g., `module_mappings` or `explicit_module_mappings`. You should also set `metavar='MODULE MAPPING'`

Reply:
Is it too long `--module-map-from-file` or `--module-maps-file` for the file of mappings? Then maybe we could have `--module-map` for the string mappings, instead of `--try-module`? What do you think?

Reply:
The problem with naming `--module-map` the current `--try-module` is that it is not a map. It simply defines a mapping.

What about `--module-mappings=FILE` for the one taking the file as argument and `--map-module=MAPPING` the one taking a mapping? I don't think is necessary to include `file` as part of the option name, because you can nicely show in the help message that this option expects  a file by setting `metavar='FILE'`.

Reply:
Having a `--module-map` and a `--map-module` option can be confusing. That's why I prefer `--module-mappings` for the first one. Your help message is fine for this option. For the second one, I find the "to the corresponding module" confusing, because I can't see where this "corresponding module" is. I'd suggest sth like "Apply a single module mapping".